apps/SageAccountWeb/src/components/SignUpdatedTermsOfUsePage.tsx

```diff
@@ -108,6 +110,8 @@ export const SignUpdatedTermsOfUsePage = (
                 onClick={() => {
                   sessionStorage.setItem('skippedSigningToS', 'true')
                   restoreLastPlace(history)
+                  //if we did not redirect to a page in this app, then look for the redirect cookie
+                  processRedirectURLInOneSage()
```

Comment:
SWC-7127: Fix the redirect issue after skipping or agreeing to the new ToS

Reply:
One minor change suggested in #1314